applications/StructuralMechanicsApplication/custom_utilities/formfinding_IO_utility.h

```diff
@@ -0,0 +1,64 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics 
+//
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Anna Rehr
+//
+
+#if !defined(KRATOS_FORMFINDING_UTILIY_H_INCLUDED )
+#define  KRATOS_FORMFINDING_UTILIY_H_INCLUDED
+
+
+
+// System includes
+#include <string>
+#include <iostream> 
+
+
+// External includes 
+
+
+// Project includes
+#include "includes/define.h"
+#include "includes/element.h"
```

Comment:
elemnt.h and properties.h are not needed if you include model_part.h

also string and iostream can be removed from what I see

Reply:
done